packages/app/src/overview/components/overview-screens/kompakt-overview/kompakt-overview.component.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
+import { KompaktOverviewProps } from "App/overview/components/overview-screens/kompakt-overview/kompakt-overview.component.interface"
+
+export const KompaktOverview: FunctionComponent<KompaktOverviewProps> = ({
+  batteryLevel = 0,
+  caseColour,
+  networkName,
+  networkLevel,
+  serialNumber,
+  osVersion,
```

Comment:
[optional] Maybe for Kompact we can use useSelector hook instead of props.

Reply:
done :)